remediation/fence-agents-remediation/known-issues.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: default
+title: Known Issues
+parent: Fence Agents Remediation
+grand_parent: Remediation
+nav_order: 4
+---
+
+## Known Issues
+
+1. FAR can't be updated from v0.1.0 to any newer versions, since there was a change in the commonLabels to one of [Kubernetes recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/) in FAR v0.2.0.
```

Comment:
The label isn't the reason. The reason is us not dealing with the known issue...
I would skip the reason at all 🤷🏼‍♂️

Reply:
SGTM the reason is not so important information for a TP release.